utility/parseCSV.py

```diff
@@ -0,0 +1,83 @@
+# Requirements: 
+# 1. Entering data spreadsheets to the function in order as referenced.
+# 2. Updating priority field on both spreadsheets, ensuring value is equal on both. 
+
+import pandas as pd
+
+# parseCSV will accept the BOM.csv and MS_Quote Excel File and return two seperate Excel/CSV files with winning parts. 
+def parseCSV(Bom, MS_Quote): 
+
+    Bom_DF = pd.read_csv(Bom)
+    MS_Quote_DF = pd.read_excel(MS_Quote)
```

Comment:
Thinking that we should check with Kristen whether we will always get first script in csv, and second in excel. Feels like something we might trip on in future uses